agent/api/serviceconnect/service_connect_attachment_parser.go

```diff
@@ -23,34 +23,32 @@ import (
 )
 
 const (
-	// serviceConnectAttachmentType specifies attachment type for service connect
-	serviceConnectAttachmentType = "ServiceConnect"
-	// serviceConnectConfigKey specifies the key maps to the service connect config in attachment properties
-	serviceConnectConfigKey = "ServiceConnectConfig"
-	// serviceConnectContainerNameKey specifies the key maps to the service connect container name in attachment properties
-	serviceConnectContainerNameKey = "ContainerName"
+	// ServiceConnectConfigKey specifies the key maps to the service connect config in attachment properties
+	ServiceConnectConfigKey = "Config"
```

Comment:
The config key is coming in from Attachment and the value is still `ServiceConnectConfig`. Why is this changed?

Reply:
Sorry that's a miss. Will fix.